pywbemtools/pywbemcli/_context_obj.py

```diff
@@ -288,88 +288,41 @@ def execute_cmd(self, cmd):
             if not self.pdb:
                 self.spinner_stop()
 
-            # Issue statistics if required. Note: uses _conn in order
-            # not to create the connection if not created.
-            if self.timestats and self._conn:
+            # Issue statistics if required.
+            if self.timestats:
                 context = click.get_current_context()
                 click.echo(self.format_statistics(self.conn.statistics,
                                                   context.obj))
 
     def format_statistics(self, statistics, context):
         # pylint: disable=no-self-use
         """
-        Table formatted output of statistics
+        Table formatted output of client statistics
         """
-        if context.output_format:
-            if not output_format_in_groups(context.output_format, 'TEXT'):
-                click.echo(statistics.formatted())
-                return
-
-        def format_int(avg, min_, max_):
-            """Display float statistics with 0 places"""
-            avg = int(avg)
-            min_ = int(min_)
-            max_ = int(max_)
-            if avg == min_ == max_:
-                return '{0}'.format(avg)
-            return '{0}/{1}/{2}'.format(avg, min_, max_)
-
-        def format_float3(avg, min_, max_):
-            """Display float statistics with 0 places"""
-            if avg == min_ == max_:
-                return '{0:.3f}'.format(avg)
-            return '{0:.3f}/{1:.3f}/{2:.3f}'.format(avg, min_, max_)
+        output_fmt = validate_output_format(context.output_format, 'TABLE')
 
         snapshot = sorted(statistics.snapshot(),
                           key=lambda item: item[1].avg_time,
                           reverse=True)
 
-        # Determine of svr_time or lengths should be included.
-        include_svr_time = False
-        include_lengths = False
-        for name, stats in snapshot:  # pylint: disable=unused-variable
-            # TODO: clean up pywbem stats so this is in pywbem, not here
-            # pylint: disable=protected-access
-            if stats._server_time_stored:  # pylint: disable=protected-access
-                include_svr_time = True
-                # pylint: disable=protected-access
-            if stats._request_len_sum > 0 or stats._reply_len_sum > 0:
-                include_lengths = True
-
-        # build list of column names
-        header = ['Op\nCnt', 'Exc\nCnt', 'Op Time(S)\nAvg/Min/Max']
-        if include_svr_time:
-            header.append("Server Time(S)\nAvg/Min/Max")
-        if include_lengths:
-            header.extend(['RequestLen\nAvg/Min/Max', 'ReplyLen\nAvg/Min/Max'])
-
-        # build table rows from snapshot of OperationStatistics
+        header = ['Operation', 'Count', 'Errors',
```

Comment:
should we compress these headers in to columns that more closely match the data wiidths and use multiple lines for the headers??? Also same question about the operation name to the lefl vs the right column,.

QUESTION: Should we consider sorting thee rows here? We could sort by names, number of operations, etc.

Reply:
On optimizing column headers: See [this comment](https://github.com/pywbem/pywbemtools/pull/907#discussion_r580009846)
On operation name left or right: See [this comment](https://github.com/pywbem/pywbemtools/pull/907#discussion_r580006287)

Now on sorting the rows:

The current code does the following:
* For the client statistics: Sorted by decreasing average client time (see `ContextObj.format_statistics()`)
* For the server statistics: Not sorted on client side, displays enumerated instances as received (see `cmd_statistics_server_show()`)

**DISCUSSION**: How to sort by default? Should we provide options for users to select sorting?

Reply:
Darn, I forgot that I did to that sort. Maybe we should sort the server stats the same way we sort the client stats. In particular, since we do not have any idea of the order received from the server it might make sense to make the order independent of the server order.

Reply:
We agreed that for now, we would sort the server statistics also by average server time.

Reply:
The latest update implements that. DONE.